Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision cb6b3e8c9b6c26ffef473afa13f2f537ce97b468)
+++ b/main.py	(date 1632814243656)
@@ -5,4 +5,5 @@
 # another
 # fdsdfdffsfdsdfsdfsdfrfrfrrrrr
 # new branch
-# another newwww
\ No newline at end of file
+# another newwww
+# another another
\ No newline at end of file
